bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/import/configDialog.js

```diff
@@ -125,6 +125,14 @@ pimcore.object.helpers.import.configDialog = Class.create({
 
         buttons.push(this.deleteButton);
 
+        this.exportConfigButton = new Ext.menu.Item({
```

Comment:
From a user experience perspective I believe that the "Load" split button is not the right place for exporting something. Maybe rephrase the entire menu? Or even combine "Load" and "Save" split button ?

Reply:
Sincerely, i'm not very skilled in terms of UX :)
I think that the "Import" button could remain in the Load split button. I can just move the "Export" behind the Save split button.
What do you think about it?

Reply:
better than now, because you will never search for an export button in a "Load" menu :-)

Reply:
Moved the Export functionality on the Save split button